addons/auth_signup/models/res_users.py

```diff
@@ -263,6 +264,10 @@ def send_unregistered_user_reminder(self, after_days=5):
             template = email_template.with_context(dbname=self._cr.dbname, invited_users=invited_users[user])
             template.send_mail(user, email_layout_xmlid='mail.mail_notification_light', force_send=False)
 
+        done = len(res_users_with_details)
+        res_users_count = 0 if done < batch_size else self.env['res.users'].search_count(domain)
+        return done, res_users_count
```

Comment:
In accordance with the former comment, we suggest to always notify explicitly as
```suggestion
        done = len(res_users_with_details)
        self.env['ir.cron']._notify_progress(
            done=done,
            remaining=0 if done < batch_size else self.env['res.users'].search_count(domain),
        )
```